vllm/README.md

```diff
@@ -0,0 +1,171 @@
+# vLLM-based Downstream Serving Image
+
+Our project focuses on downstream applications built on vLLM, providing a Docker image with features tailored to internal customers' needs as well as support for advanced features.
+
+---
+
+## Table of Contents
+
+1. [Getting Started and Usage](#1-getting-started-and-usage)  
+   1.1 [Pulling and Running the Docker Container](#11-pulling-and-running-the-docker-container)  
+   1.2 [Launching the Serving Service](#12-launching-the-serving-service)  
+   1.3 [Benchmarking the Service](#13-benchmarking-the-service)  
+2. [Advanced Features](#2-advanced-features)  
+   2.1 [CCL Support (both P2P & USM)](#21-ccl-support-both-p2p--usm)  
+   2.2 [INT4 and FP8 Quantized Online Serving](#22-int4-and-fp8-quantized-online-serving)  
+   2.3 [Model++: Embedding and Reranker Support](#23-model-embedding-and-reranker-support)  
```

Comment:
2.2 Embedding and Reranker Model Support
2.3 Multi-Modal Model Support
2.4 Data Parallelism (DP)